### PR TITLE
feat: use updated version of aleph-pytezos

### PR DESCRIPTION
**Pull request by Psycojoker**

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -41,7 +41,7 @@ ethereum = ["eth_account>=0.4.0"]
 polkadot = ["substrate-interface==1.3.4"]
 cosmos = ["cosmospy==6.0.0"]
 solana = ["pynacl==1.5.0", "base58==2.1.1"]
-tezos = ["pynacl==1.5.0", "aleph-pytezos==0.1.0"]
+tezos = ["pynacl==1.5.0", "aleph-pytezos@git+https://github.com/aleph-im/aleph-pytezos.git@main"]
```
Comment (hoh) [inline pyproject.toml:44]: We cannot publish `aleph-client` on PyPI if all dependencies are not on PyPI.
```suggestion
tezos = ["pynacl==1.5.0", "aleph-pytezos==3.13.4"]
```

Comment (Psycojoker) [reply pyproject.toml:44]: Ah, forget to push, this was already done, sowwy

**Comment by hoh** [comment]
CI is failing due to Mypy.

**Comment by Psycojoker** [comment]
Yes, it's also failing on main/master so I didn't paid attention here since it's unrelated

Which is surprising because the PR related to this commit was green https://github.com/aleph-im/aleph-client/pull/259 

image

**Review by hoh — APPROVED** [review]

**Pull request merged**